README.md

```diff
@@ -60,6 +60,7 @@ name
 [explicit_counter_loop](https://github.com/Manishearth/rust-clippy/wiki#explicit_counter_loop)                       | warn    | for-looping with an explicit counter when `_.enumerate()` would do
 [explicit_iter_loop](https://github.com/Manishearth/rust-clippy/wiki#explicit_iter_loop)                             | warn    | for-looping over `_.iter()` or `_.iter_mut()` when `&_` or `&mut _` would do
 [extend_from_slice](https://github.com/Manishearth/rust-clippy/wiki#extend_from_slice)                               | warn    | `.extend_from_slice(_)` is a faster way to extend a Vec by a slice
+[filter_map](https://github.com/Manishearth/rust-clippy/wiki#filter_map)                                             | allow   | using `filter(_).map(_)`, which is more succinctly expressed as `.filter_map(_)`
```

Comment:
Perhaps the description should note the other different patterns the (now unified) lint finds?